stone/backends/python_rsrc/stone_validators.py

```diff
@@ -69,6 +69,11 @@ def __repr__(self):
         return 'ValidationError(%r)' % six.text_type(self)
 
 
+def type_name_with_module(t):
```

Comment:
shouldn't we always use `generic_type_name` instead of this right?

Reply:
I don't think that `generic_type_name` will work for the `self.definition` case later on - it'll end up returning `builtins.type`.